src/main/kotlin/org/dif/peerdid/PeerDIDCreator.kt

```diff
@@ -16,22 +21,46 @@ fun isPeerDID(peerDID: String): Boolean {
     return regex.matches(peerDID)
 }
 
-/** Creates [PeerDID] according to zero algorithm using [inceptionKey]*/
+/**
+ * Generates PeerDID according to the zero algorithm
```

Comment:
It would be great to add references to the peer did spec (peer did regex, method0 and method2)